nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -527,6 +529,8 @@
 
 - `networking.wireguard` now has an optional networkd backend. It is enabled by default when `networking.useNetworkd` is enabled, and it can be enabled alongside scripted networking with `networking.wireguard.useNetworkd`. Some `networking.wireguard` options have slightly different behavior with the networkd and script-based backends, documented in each option.
 
+- `networking.wireguard` adds support for the [AmneziaWG](https://amnezia.org/en) variant of the protocol. The variant to be used is set per interface as `networking.wireguard.interfaces.<name>.type`, defaulting to wireguard.
```

Comment:
There's also the new module `programs.amnezia-vpn` introduced in this release. Can @averyanalex or @sund3RRR outline the differences of these approaches of using and Amnezia VPN? I'd like to to make this more clear in the release notes.

Reply:
`programs.amnezia-vpn` configures GUI client with imperative connections management, while `networking.wireguard` - declarative management without GUI